packages/shared/lib/core/wallet/utils/transactions/activityOutputContainsValue.ts

```diff
@@ -0,0 +1,22 @@
+import { ActivityType } from '@core/wallet/enums'
+import { IBasicOutput } from '@iota/types'
+import { getNativeTokenFromOutput, isOutputAsync } from '..'
+import { IWrappedOutput } from '../../interfaces'
+import { getActivityTypeFromOutput, getAmountFromOutput, getStorageDepositFromOutput } from '../generateActivity/helper'
+
+export function activityOutputContainsValue(wrappedOutput: IWrappedOutput): boolean {
+    const type = getActivityTypeFromOutput(wrappedOutput)
+    const typesToCheck = [ActivityType.Basic]
+    if (typesToCheck.includes(type)) {
```

Comment:
Why is `ActivityType.Basic` assigned as a single element array and then searched with `includes(type)` when `ActivityType.Basic === type` would return the same? Will we eventually add more elements into `typesToCheck`?